### PR TITLE
Fixed an error when using the command to download src

### LINKED ISSUE #2938: Download source code to file error when using fission command
<pre>
$ fission version
client:
  fission/core:
    BuildDate: "2024-01-14T15:43:35Z"
    GitCommit: 7e8d5dd7
    Version: v1.20.1
server:
  fission/core:
    BuildDate: "2024-01-14T15:43:35Z"
    GitCommit: 7e8d5dd7
    Version: v1.20.1
</pre>

**cmd**
fission pkg getsrc --name kafka-zip-czub  --output=/home/123

**error occurred**
image

**Describe the bug**
An error occurred while using the command download

### PR DESCRIPTION
**Pull request by waitstory**
issue: #2938

## Description
Fixed an error when using the command to download src

## Which issue(s) this PR fixes:

Fixes #2938

## Checklist:

- [ ] I ran tests as well as code linting locally to verify my changes. 
- [ ] I have done manual verification of my changes, changes working as expected.

### CODE REVIEW

**Comment by sanketsudake** [comment]
Just following https://github.com/golang/go/issues/41487 so see if there is better way to handle this issue.

**Comment by waitstory** [comment]
> Just following [golang/go#41487](https://github.com/golang/go/issues/41487) so see if there is better way to handle this issue.就按照 [golang/go#41487](https://github.com/golang/go/issues/41487) 看看是否有更好的方法来处理这个问题。

There seems to be no direct solution to the problem

File: pkg/fission-cli/cmd/package/util/util.go [hunk 1]
```diff
@@ -217,7 +218,8 @@ func WriteArchiveToFile(fileName string, reader io.Reader) error {
 		return err
```
[the remark this answers is not in the document]
Comment (waitstory) [reply pkg/fission-cli/cmd/package/util/util.go:218]: Seems like a good idea

**Comment by sanketsudake** [comment]
@waitstory Would you like to make the above change or should I raise a separate PR?

**Comment by waitstory** [comment]
Let me make the change and submit it

**Review by sanketsudake — APPROVED** [review]

**Pull request merged**